internal/stack/_static/kibana.yml.tmpl

```diff
@@ -161,7 +161,3 @@ xpack.fleet.outputs:
 {{- if eq $version "9.0.0-SNAPSHOT" }}
 xpack.fleet.internal.registry.kibanaVersionCheckEnabled: false
 {{- end }}
-
-logging.loggers:
-  - name: plugins.fleet
-    level: debug
```

Comment:
Reverting the change introduced in https://github.com/elastic/elastic-package/pull/2501/ related to the log level of fleet plugin.